scripts/brp-python-bytecompile

```diff
@@ -15,14 +24,60 @@ if [ -z "$depth" ] || [ "$depth" -le "1" ]; then
 	exit 0
 fi
 
+# This function now implements Python byte-compilation in three different ways:
+# Python >= 3.4 and < 3.9 uses a new module compileall2 - https://github.com/fedora-python/compileall2
```

Comment:
We need to somehow declare a dependency on `compileall2`.

Reply:
We can drop the compileall2 thing entirely and call that a pro-active downstream only thing in Fedora, compileall2 has been merged to mainline comipleall. That would also solve the `/usr/lib/rpm/redhat/` thing.

Reply:
That works for me.

Reply:
How much of 6d322d00 and 2c5fba02 should be kept?

Reply:
I suggest we drop both of those commit entirely, but keep the Python 3.10+ branch.